lms/djangoapps/instructor/views/data_access.py

```diff
@@ -49,6 +49,19 @@ def delete_bulk_temporary_queries(query_to_delete):
     saved_students.delete()
     temp_query.delete()
 
+def delete_group_temp_queries_and_students(group_id):
+    """
+    Deletes students and active subqueries associated with an in-flight group query for efficiency.
```

Comment:
When I call this in bulk email, the group query will have finished already. This seems confusing, wouldn't it be dangerous to delete stuff for a query that is in-flight?